test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/internal/WebView/TSWebCookieManager.cs

```diff
@@ -14,25 +15,71 @@ namespace Tizen.NUI.Devel.Tests
     public class InternalWebCookieManagerTest
     {
         private const string tag = "NUITEST";
-        private static string[] runtimeArgs = { "Tizen.NUI.Devel.Tests", "--enable-dali-window", "--enable-spatial-navigation" };
-        private const string USER_AGENT = "Mozilla/5.0 (SMART-TV; Linux; Tizen 6.0) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/4.0 Chrome/76.0.3809.146 TV Safari/537.36";
+        private string urlForCookies = "https://www.baidu.com/";
+        private BaseComponents.WebView _webview;
+        private WebCookieManager _cookieManager;
+        private const string cookiePath = "/home/owner/.cookie";
 
         [SetUp]
         public void Init()
         {
             tlog.Info(tag, "Init() is called!");
+            _webview = new BaseComponents.WebView()
```

Comment:
if possible, from next time, please don't use "_" as for private member. (as I know, it is old style ^^;)

Reply:
Done